docs/tutorials/word2vec.ipynb

```diff
@@ -37,20 +37,20 @@
         "id": "AOpGoE2T-YXS"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.        for i in range(window_size, len(sequence)-window_size):
</pre><p>You may as well build this dict from the existing positive_skip_grams, rather than iterating over the window from scratch.  You could also use a <code>set</code> to avoid duplicate entries. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/text/pull/1229/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>

Reply:
<p>Nice idea, I'll try out something like this:</p><pre class="ql-syntax" spellcheck="false">window = defaultdict(set)
for target_word, context_word in positive_skip_grams:
  window[target_word].add(context_word)
  ...
</pre>